src/Microsoft.Diagnostics.Monitoring.EventPipe/Counters/CounterPayload.cs

```diff
@@ -182,17 +250,36 @@ public PercentilePayload(CounterMetadata counterMetadata, string displayName, st
     // dotnet-counters created a separate payload for each quantile (multiple payloads per TraceEvent).
     // AggregatePercentilePayload allows dotnet-monitor to construct a PercentilePayload for individual quantiles
     // like dotnet-counters, while still keeping the quantiles together as a unit.
-    internal sealed class AggregatePercentilePayload : MeterPayload
+    internal sealed class AggregatePercentilePayload : MeterInstrumentDeltaMeasurementPayload
     {
-        public AggregatePercentilePayload(CounterMetadata counterMetadata, string displayName, string displayUnits, string valueTags, IEnumerable<Quantile> quantiles, DateTime timestamp) :
+        public AggregatePercentilePayload(CounterMetadata counterMetadata, string displayName, string displayUnits, string valueTags, int count, double sum, IEnumerable<Quantile> quantiles, DateTime timestamp) :
             base(timestamp, counterMetadata, displayName, displayUnits, 0.0, CounterType.Metric, valueTags, EventType.Histogram)
         {
+            Count = count;
+            Sum = sum;
             //string counterName = string.IsNullOrEmpty(displayName) ? name : displayName;
             //DisplayName = !string.IsNullOrEmpty(displayUnits) ? $"{counterName} ({displayUnits})" : counterName;
             Quantiles = quantiles.ToArray();
         }
 
-        public Quantile[] Quantiles { get; }
+        public int Count { get; private set; }
+
+        public double Sum { get; private set; }
+
+        public Quantile[] Quantiles { get; private set; }
+
+        public override bool SupportsDelta => true;
```

Comment:
Does doing delta aggregation make sense given that the quantiles can't be combined? I see that you are only doing it for Count and Sum and those do work functionally, so this is more of a broader scenario question if it makes sense to offer this partial information for Histograms vs leaving them out for now and telling people they have to update to a newer version of .NET to make that part of the scenario work.

Reply:
The whole combine concept is gone for now. I left count/sum on here though. Talking with @alanwest backends like NR can do nice things with just sum/count so I felt it was still worth exposing the data to interested listeners 🤷